src/mixins/input-styles/_input-styles.scss

```diff
@@ -1,6 +1,11 @@
 @mixin input-styles() {
   @include text(body1);
 
+  //ios devices zoom in if inputs are smaller than 16px
+  @include breakpoint(sm, $prop: max-width) {
+    font-size: 16px;
```

Comment:
You'd probably want to use `@include text(h6);` or at the very least `font-size: __dangerously-get-font-size__(2);` (setting it to 1rem, under control of the design system/tokens, rather than hardcoding a px value.